tensorboard/manager.py

```diff
@@ -199,3 +203,100 @@ def cache_key(working_directory, arguments, configure_kwargs):
   # `raw` is of type `bytes`, even though it only contains ASCII
   # characters; we want it to be `str` in both Python 2 and 3.
   return str(raw.decode("ascii"))
+
+
+def _get_info_dir():
+  """Get path to directory in which to store info files.
+
+  The directory returned by this function is "owned" by this module. If
+  the contents of the directory are modified other than via the public
+  functions of this module, subsequent behavior is undefined.
+
+  The directory will be created if it does not exist.
+  """
+  path = os.path.join(tempfile.gettempdir(), ".tensorboard-info")
+  try:
+    os.makedirs(path)
+  except OSError as e:
+    if e.errno == errno.EEXIST and os.path.isdir(path):
+      pass
+    else:
+      raise
+  return path
+
+
+def _get_info_file_path():
+  """Get path to info file for the current process."""
```

Comment:
May want to note that this will create the info directory if it doesn't exist, so it's not a pure getter.

Reply:
Sure.